src/app/test-record/defects/defects.component.ts

```diff
@@ -0,0 +1,22 @@
+import {Component, OnInit, ChangeDetectionStrategy, Input} from '@angular/core';
+import {TestType} from '@app/models/test.type';
+import {TestResultModel} from '@app/models/test-result.model';
+
+@Component({
+  selector: 'vtm-defects',
+  templateUrl: './defects.component.html',
+  styleUrls: ['./defects.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class DefectsComponent implements OnInit {
+
+  @Input() testType: TestType;
```

Comment:
you should be able to write test to mock these @Input(). Please check the ADR branch or lets talk

Reply:
![image](https://user-images.githubusercontent.com/55185857/77905777-89711200-728f-11ea-8c36-aa1a61ef3b95.png)

Reply:
![image](https://user-images.githubusercontent.com/55185857/77905847-a279c300-728f-11ea-8006-ce9f8b4bfecf.png)